src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml

```diff
@@ -62,18 +64,18 @@
         </Path>
 
         <!--This Polygon will draw on the Canvas the pointer (a triangle) that will be available only for tooltips but for the Welcome popup will be hidden-->
-        <Polygon x:Name="TooltipPointer"
-                 Visibility="{Binding ShowPopupPointer, Converter={StaticResource BooleanToVisibilityConverter}}"
-                 Points="{Binding TooltipPointerPoints}" 
-                 Style="{StaticResource PoupPolygonPointerStyle}">
-            <Polygon.BitmapEffect>
-                <!--This effect will show a 4px shadow of 20% of tranparency with a angle of 315 grades by default-->
-                <DropShadowBitmapEffect Color="Black" 
-                                        Direction="135" 
-                                        ShadowDepth="4" 
-                                        Opacity="0.2" />
-            </Polygon.BitmapEffect>
-        </Polygon>
+        <Path  x:Name="TooltipPointer"
```

Comment:
Can we update the comment above as well?

Reply:
@QilongTang comment fixed
https://github.com/DynamoDS/Dynamo/pull/12503/commits/d52c3d84aa241fd5f5dd0df0d24b8c060db43d7c